image/db/rhel/Dockerfile

```diff
@@ -25,18 +25,30 @@ COPY --from=extracted_bundle /bundle/etc/postgresql.conf /bundle/etc/pg_hba.conf
 
 ARG POSTGRESQL_ARCH=x86_64
 
-RUN groupadd -g 70 postgres && \
+RUN microdnf install -y shadow-utils && \
+    groupadd -g 70 postgres && \
     adduser postgres -u 70 -g 70 -d /var/lib/postgresql -s /bin/sh && \
-    dnf upgrade -y && \
-    dnf install -y \
+    curl -sSLf https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-${PG_MAJOR} -o /tmp/pg_repo.key && \
+    rpm --import /tmp/pg_repo.key && \
+    curl -sSLf https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-AARCH64 -o /tmp/pg_repo.key && \
+    rpm --import /tmp/pg_repo.key && \
```

Comment:
Why do we need both of these?

Reply:
From the looks of it, I was able to succeed with just the first one:
```
[root@f9e6d3a223f7 /]# curl -sSLf https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-12 -o /tmp/pg_repo.key
[root@f9e6d3a223f7 /]# rpm --import /tmp/pg_repo.key
[root@f9e6d3a223f7 /]# curl -sSLf https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm -o /tmp/pg_repo.rpm
[root@f9e6d3a223f7 /]# rpm -i /tmp/pg_repo.rpm
[root@f9e6d3a223f7 /]# rpm -qa | grep pgdg
pgdg-redhat-repo-42.0-27.noarch
```

Reply:
The second one is for arm. But on my M1 Mac the GPG check was failing even with it.